rllib/tests/test_io.py

```diff
@@ -228,17 +226,13 @@ def test_multi_agent(self):
                 },
             )
             pg.train()
-            self.assertEqual(len(os.listdir(self.test_dir)), 1)
-
+            self.assertEqual(len(os.listdir(self.test_dir + fw)), 1)
             pg.stop()
             pg = PG(
                 env="multi_agent_cartpole",
                 config={
                     "num_workers": 0,
-                    "input": self.test_dir,
-                    "off_policy_estimation_methods": {
-                        "simulation": {"type": "simulation"}
-                    },
+                    "input": self.test_dir + fw,
```

Comment:
Quick question: We still support both a) a dir string (read all JSON files in the dir) and b) a list of filenames, correct?